ismatec/mock.py

```diff
@@ -23,8 +27,8 @@ def __init__(self, *args, **kwargs):
         self.channels = [1, 2, 3, 4]
         self.running = {channel: False for channel in self.channels}
         self.state = {
-            'channel_addressing': False,  # FIXME verify
-            'event_messaging': False,  # FIXME verify
```

Comment:
Unknown what the fixme is for.

Reply:
The fixme is for verifying the initial values on a reset / powered up pump.